src/DynamoCore/Graph/Nodes/NodeModelExtensions.cs

```diff
@@ -101,9 +101,12 @@ internal static bool IsUpstreamOf(this NodeModel node, NodeModel otherNode)
 
         internal static List<IGraphicItem> GeneratedGraphicItems(this NodeModel node, EngineController engineController)
         {
-            var ids = node.GetAllOutportAstIdentifiers();
-
             var results = new List<IGraphicItem>();
+            if (node is null || engineController is null)
+            {
+                return results;
+            }
+            var ids = node.GetAllOutportAstIdentifiers();
```

Comment:
EngineController looks like the main suspect
https://git.autodesk.com/Dynamo/DynamoRevit/blob/14cf7520884a2e285c81241847097fccf1cd6999/src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs#L121-L133

And I do see a lot of code in Dynamo checks if the controller is null (maybe some graphics are generated while the DynamoModel is being disposed ?...)